val/sys_arch_src/smmu_v3/smmu_v3.c

```diff
@@ -853,7 +853,7 @@ static uint32_t smmu_probe(smmu_dev_t *smmu)
     smmu->ias = get_max(smmu->ias, smmu->oas);
 
     val_print(ACS_PRINT_INFO, "  ias %d-bit ", smmu->ias);
```

Comment:
Could you drop spaces before "ias"? Will align which "oas" from next line.

Reply:
Hi @hrw,
We expect ias and oas to be on the same line on the console. The -v 1 log would look something like this -
  ... 
  Initializing SMMU
  ssid_bits = 20 sid_bits = 24
  ias 52-bit  oas 52-bit
  ssid_bits = 20 sid_bits = 24
  ias 52-bit  oas 52-bit
  Initializing ITS
  ITS : Info Block
  ...

Thanks,
ACS team.

Reply:
Ah, sorry then.